src/main/webapp/app/course/tutorial-groups/tutorial-groups-management/tutorial-groups/tutorial-groups-management/tutorial-groups-import-dialog/tutorial-groups-registration-import-dialog.component.html

```diff
@@ -30,6 +30,94 @@ <h4 class="modal-title">
                 </ul>
             </ul>
         </div>
+        <!-- Example Tables -->
+        <div class="form-group">
+            <label class="font-weight-bold pointer" (click)="generateCSV(1)">
+                <th>{{ 'artemisApp.tutorialGroupImportDialog.table.example1' | artemisTranslate }}</th>
```

Comment:
We prefer jhiTranslate over artemisTranslate (optional)

Reply:
changed it 👍